Xamarin.Forms.Controls/GalleryPages/DateTimePickerGalleries/DatePage.xaml.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Xamarin.Forms;
+
+namespace Xamarin.Forms.Controls.GalleryPages.DateTimePickerGalleries
+{
+	public partial class DatePage : ContentPage
+	{
+		public DatePage()
+		{
+			InitializeComponent();
+			BindingContext = this;
+			var dep = DependencyService.Get<ILocalize>();
+			if (dep != null)
+			{
+				datesphoneculture.Text = "Device Culture: " + dep.GetCurrentCultureInfo();
```

Comment:
Not a necessary change but another way to do this would be
`datesphoneculture.Text = $"Device Culture: {dep.GetCurrentCultureInfo()}";`